ui/app/components/ui/list-item/list-item.component.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Approve from '../icon/approve-icon.component'
+import Interaction from '../icon/interaction-icon.component'
+import Preloader from '../icon/preloader'
+import Send from '../icon/send-icon.component'
+import classnames from 'classnames'
+
+const SendIcon = () => (
+  <Send
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const InteractionIcon = () => (
+  <Interaction
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const ApproveIcon = () => (
+  <Approve
+    className="list-item__icon"
+    size={28}
+    color="#2F80ED"
+  />
+)
+
+const FailIcon = () => (
+  <Interaction
+    className="list-item__icon"
+    size={28}
+    color="#D73A49"
+  />
+)
+
+const Item = ({
+  className,
+  status,
+  title,
+  subtitle,
+  children,
+  primaryCurrency,
+  secondaryCurrency,
+}) => {
+  const isApproved = status === 'approved'
+  const isUnapproved = status === 'unapproved'
+  const isPending = status === 'pending'
+  const isFailed = status === 'failed'
+
+  let icon = <InteractionIcon />
+  if (isApproved) {
+    icon = <ApproveIcon />
+  } else if (isPending) {
+    icon = <SendIcon />
+  } else if (isFailed) {
+    icon = <FailIcon />
+  }
+
+  let subtitleStatus = null
+  if (isUnapproved) {
+    subtitleStatus = (
+      <span><span className="list-item__status--unapproved">Unapproved</span> · </span>
```

Comment:
Ooh, one more small change needed here: translations for "Unapproved" and "Failed"